pkg/sql/pg_catalog.go

```diff
@@ -481,6 +483,8 @@ CREATE TABLE pg_catalog.pg_attribute (
 				tree.DNull, // attalign
 				tree.MakeDBool(tree.DBool(!column.Nullable)),          // attnotnull
 				tree.MakeDBool(tree.DBool(column.DefaultExpr != nil)), // atthasdef
+				tree.DNull,         // attidentity
+				tree.DNull,         // attgenerated
```

Comment:
would a computed column be a generated column?

Reply:
Yeah it would be, my bad here I told @r6min not to worry about it in this PR, wasn't sure whether to actually populate the columns in this PR but I think it would make sense to, it should be a fairly easy add.

Reply:
Yep, I can implement `attgenerated` if needed.

Reply:
if it unblocks people immediately i'd say go for it as is, but file an issue.
if it doesn't take too much time (i think it shouldn't but could be wrong), i'd say may as well do it now.

Reply:
also reading the docs, this should be a zero byte (`\0`) as opposed to `tree.DNull`?

Reply:
Yep, should be a quick fix. I'll set `attgenerated` to `\0` if it isn't a computed column.